### PR TITLE
Update ssh_autodetect.py

### PR DESCRIPTION
**Pull request by rafaelesmundo**
added extreme boss and voss switching platform for autodetect ssh. Updated in the SSH_MAPPER_DICT.

### CODE REVIEW

**Comment by ktbyers** [comment]
@rafaelesmundo Neither of these drivers exist in Netmiko?

```
    "extreme_voss"
    "extreme_boss"
```

**Comment by lmesserStep** [comment]
@ktbyers Extreme_boss should be extreme_ers_ssh.py and Extreme_Voss should be extreme_vsp_ssh.py

File: netmiko/ssh_autodetect.py [hunk 1]
```diff
@@ -204,6 +204,18 @@
         "priority": 99,
         "dispatch": "_autodetect_std",
     },
+    "extreme_voss": {
```
Comment (lmesserStep) [inline netmiko/ssh_autodetect.py:207]: extreme_voss needs to be extreme_vsp

**Comment by ktbyers** [comment]
@lmesserStep Do you want to fix the PR (otherwise, it just means I have to fix it :-)

**Comment by ktbyers** [comment]
Closing as the PR isn't correct.

**Pull request closed without merging**